fiasko_bro/code_helpers.py

```diff
@@ -41,3 +42,14 @@ def get_mccabe_violations_for_file(filepath, max_complexity):
 def count_indentation_spaces(line, tab_size=4):
     expanded_line = line.expandtabs(tab_size)
     return len(line) - len(expanded_line.lstrip())
+
+
+def is_repo_too_large(path_to_repo, max_py_files_count, path_to_original_repo=None):
+    num_of_py_files = count_py_files(path_to_repo)
+    if num_of_py_files > max_py_files_count:
+        return True
+    if path_to_original_repo:
+        num_of_py_files_original_repo = count_py_files(path_to_original_repo)
+        return num_of_py_files_original_repo > max_py_files_count
+
```

Comment:
Here an in `test_repo_size.py`: don't put the additional blank line.
1) It's inconsistent with the existing code;
2) PEP8 [asks](https://www.python.org/dev/peps/pep-0008/#blank-lines) you use them sparingly inside the function